scripts/winesap-wsl.ps1

```diff
@@ -0,0 +1,16 @@
+$installPath = "$env:LOCALAPPDATA\Packages\winesapos"
+$winesaposTarballUrl = "https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/_test/winesapos-4.1.0-beta.0-minimal-rootfs.tar.zst"
```

Comment:
The stable version will be uploaded by tonight (I'm currently uploading the secure image first). The URL should be: `https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/winesapos-4.1.0-minimal-rootfs.tar.zst`

One problem is that this needs to be decompressed. I have to keep this compressed because my storage is limited and it helps to bandwidth for folks with data caps. Since Windows does not natively support Zstandard compression, we could (1) download the `zstd-v1.5.6-win64.zip` archive, (2) extract the ZIP, and (3) use the included `zstd.exe` executable to decompress the tarball.

https://github.com/facebook/zstd/releases

Reply:
Once merged please document the `winesap-wsl` powershell script